cms/djangoapps/contentstore/views/item.py

```diff
@@ -675,6 +675,21 @@ def _get_source_index(source_usage_key, source_parent):
         return None
 
 
+def is_source_item_in_target_parents(source_item, target_parent):
+    """
+    Returns True if source item is found in target parents otherwise False.
+
+    Arguments:
+        source_item (XBlock): Source xblock.
+        target_parent (XBlock): Target XBlock.
+    """
+    target_ancestors = _create_xblock_ancestor_info(target_parent, is_concise=True)['ancestors']
```

Comment:
Even if the `target_parent` is a course xblock or (having no parent), this `_create_xblock_ancestor_info` returns `{'ancestors': []}` which wouldn't break the code.

However, this can break if `target_parent` is `None` which would break already built method `_create_xblock_ancestor_info` because the `None` xblock wouldn't have any location or any other xblock related fields. But, I think, we are fin in this case, we would want the code to break if None is passed on (which in our scenario would never be the case).